export: Command not found.
export: Command not found.
export: Command not found.
export: Command not found.
Badly placed ()'s.
 
Running threaded PSI4 on node3 
/home/mdavis/anaconda3/envs/p4env/bin/psi4
 
Working and scratch directories
/home/mdavis/f12eom_optimizer/tests/slurm
/scratch/mdavis/PSI4_2022-08-18_11:59:18.987638679
 
use -n:  to specify number of threads
use -i:  to specify input  file     (default = input.dat)
use -o:  to specify output file     (default = file.out )
 
-h or --help for more options
 
Running...
Traceback (most recent call last):
  File "/home/mdavis/f12eom_optimizer/f12eom_optimizer/f12eom_optimizer.py", line 282, in <module>
    main()
  File "/home/mdavis/f12eom_optimizer/f12eom_optimizer/f12eom_optimizer.py", line 222, in main
    psi_gradient, psi_energy = run_psi4(molsys_obj.molsys_to_qc_molecule())
  File "/home/mdavis/f12eom_optimizer/f12eom_optimizer/f12eom_optimizer.py", line 108, in run_psi4
    gradient = retG.return_result.flatten()
AttributeError: 'FailedOperation' object has no attribute 'return_result'
...Done
 
